scripts/mixin/actuallyadditions.zs

```diff
@@ -0,0 +1,191 @@
+    }
+}
+
+#mixin Mixin
+#{targets: "de.ellpeck.actuallyadditions.mod.items.lens.LensMining"}
+zenClass MixinLensMining {
+    #mixin Static
+    #mixin Redirect
+    #{
+    #    method: "init",
+    #    at: {
+    #        value: "INVOKE",
+    #        target: "Lde/ellpeck/actuallyadditions/api/ActuallyAdditionsAPI;addMiningLensStoneOre(Ljava/lang/String;I)V"
+    #    },
+    #    slice: {
+    #       to: {
+    #           value: "INVOKE",
+    #           target: "Lde/ellpeck/actuallyadditions/mod/config/values/ConfigStringListValues;getValue()[Ljava/lang/String;"
+    #       }
+    #    }
+    #}
+    function removeRegisteredOres(name as string, weight as int) as void {
+        // NO-OP
+    }
+
+    #mixin Static
+    #mixin Redirect
+    #{
+    #    method: "init",
+    #    at: {
+    #        value: "INVOKE",
+    #        target: "Lde/ellpeck/actuallyadditions/api/ActuallyAdditionsAPI;addMiningLensNetherOre(Ljava/lang/String;I)V"
+    #    },
+    #    slice: {
+    #       to: {
+    #           value: "INVOKE",
+    #           target: "Lde/ellpeck/actuallyadditions/mod/config/values/ConfigStringListValues;getValue()[Ljava/lang/String;"
+    #       }
+    #    }
+    #}
+    function removeRegisteredNetherOres(name as string, weight as int) as void {
+        // NO-OP
+    }
+
+    #mixin Redirect
+    #{
+    #    method: "invoke",
+    #    at: {
+    #        value: "FIELD",
+    #        target: "Lde/ellpeck/actuallyadditions/api/ActuallyAdditionsAPI;STONE_ORES:Ljava/util/List;",
+    #        opcode: 178
+    #    }
+    #}
+    function removeStoneTransformation() as [string] {
+        return null;
+    }
+
+    #mixin ModifyVariable
+    #{
+    #    method: "invoke",
+    #    at: {
+    #       value: "JUMP",
+    #       opcode: 198,
+    #       ordinal: 0,
+    #       shift: "BEFORE"
+    #    },
+    #    name: "ores"
+    #}
+    function setTransformationOresIfBlockIsEndstone(ores as [WeightedOre], hitState as IBlockState, hitPos as BlockPos, tile as IAtomicReconstructor) as [WeightedOre] {
+        if (hitState.block == Blocks.END_STONE) {
+            return [
+                WeightedOre("oreEndQuartz", 64),
+                WeightedOre("oreEndAluminum", 40),
+                WeightedOre("oreEndIron", 30),
+                WeightedOre("oreEndCobalt", 20),
+                WeightedOre("oreEndArdite", 20),
+                WeightedOre("oreEndDiamond", 10)
+            ] as [WeightedOre];
+        }
+        return ores;
+    }
+
+    #mixin ModifyVariable
+    #{
+    #    method: "invoke",
+    #    at: {
+    #       value: "JUMP",
+    #       opcode: 198,
+    #       ordinal: 0
+    #    },
+    #    name: "adaptedUse"
+    #}
+    function setAdaptedUseIfBlockIsEndstone(value as int, hitState as IBlockState, hitPos as BlockPos, tile as IAtomicReconstructor) as int {
+        if (hitState.block == Blocks.END_STONE) {
+            return value + 10000;
+        }
+        return value;
+    }
```

Comment:
Please, replace this mixin with plain change of consume energy to `150000`.
Since i dont need this all AA's "adaptive cost" calculations, the static consumption will suffice. There is a line in file `de/ellpeck/actuallyadditions/mod/items/lens/LensMining`.
```diff
- tile.extractEnergy(adaptedUse);
+ tile.extractEnergy(150000);
```

Reply:
also need
```diff
- if (tile.getEnergy() >= adaptedUse)
+ if (tile.getEnergy() >= 150000)
```
So setting adaptedUse variable to 150000 is the simplest.